### PR TITLE
Documentation updates

### PR DESCRIPTION
**Pull request by micahellison**
Goals for this pr:

- [X] Apply changes from reviews of previous doc PRs - #980, #981, #982
- [x] Ensure that documentation is completely aligned with `jrnl --help` now that #1027 is merged

### Checklist

- [x] I have read the [contributing doc](https://github.com/jrnl-org/jrnl/blob/develop/CONTRIBUTING.md).
- [x] I have included a link to the relevant issue number.
- [x] I have tested this code locally.
- [x] I have checked to ensure there aren't other open [pull requests](../pulls)
  for the same issue.
- [ ] I have written new tests for these changes, as needed.
- [x] All tests pass.

### CODE REVIEW

File: docs/overview.md [hunk 1]
```diff
@@ -51,8 +48,8 @@ be [exported](./export.md) in a variety of formats.
 
 ## Multi-Platform Support
 
-`jrnl` is compatible with most operating systems. Pre-compiled binaries are
-available through several distribution channels, and you can build from source.
+`jrnl` is compatible with most operating systems. You can download it using one
```
Comment (wren) [inline docs/overview.md]: Should this link to the installation page?

Comment (micahellison) [reply docs/overview.md]: Yes, definitely!

File: docs/import-and-export.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+<!--
+@TODO: Update this for v2.5
+-->
+
+## Export to files
+
+You can specify the output file of your exported journal using the
+`-o` argument
```
Comment (wren) [inline docs/import-and-export.md]: This isn't true anymore. `-o` will still work, but only for backward compatibility. They should use `--file` like in `formats.md`.

Comment (wren) [reply docs/import-and-export.md]: Actually, do we still need this file? Maybe `formats.md` is enough to cover this?

File: docs/security.md [hunk 3]
```diff
@@ -1,65 +1,80 @@
...
 
+To delete existing `jrnl` commands from `bash` history, simply delete them from
+your bash history file. The default location of this file is `~/.bash_history`,
+but you can run `echo "$HISTFILE"` to find it if needed.  Also, you can run
+`history -c` to delete all commands from your history.
+
 ### zsh
 
-Disable history logging by adding this to your `zshrc`:
+You can disable history logging for jnrl by adding this to your `~/.zshrc`
```
Comment (wren) [inline docs/security.md]:
```suggestion
You can disable history logging for jrnl by adding this to your `~/.zshrc`
```

File: docs/encryption.md [hunk 4]
```diff
@@ -52,7 +35,7 @@ encrypted file untouched and create a new plain text file next to it.
 
 ## Storing Passwords in Your Keychain
 
-There is no method to recover or reset your `jrnl` password. If you lose it,
+You can't recover or reset your `jrnl` password. If you lose it,
```
Comment (wren) [inline docs/encryption.md]: Maybe "nobody can" could work here? It's also fine as-is.

File: docs/recipes.md [hunk 5]
```diff
@@ -113,7 +113,7 @@ logged as a new entry in the journal you specified in the original argument.
 
 !!! tip 
     To read your journal entry or to verify the entry saved, you can use this 
-    command: `jrnl -n 1` (Check out [Import and Export](./export.md) for more export options).
+    command: `jrnl -n 1` (Check out [Formats](./formats.md) for more export options).
```
Comment (wren) [inline docs/recipes.md]:
```suggestion
    command: `jrnl -n 1` (Check out [Formats](./formats.md) for more options).
```

File: mkdocs.yml [hunk 6]
```diff
@@ -19,6 +19,7 @@ nav:
   - Basic Usage: usage.md
   - Encryption: encryption.md
   - Privacy and Security: security.md
-  - Import and Export: export.md
+  - Formats: formats.md
+  - Import and Export: import-and-export.md
```
Comment (wren) [inline mkdocs.yml]: If we remove the export file above, we should take it out of here, too.

**Review by wren — CHANGES_REQUESTED** [review]
This is looking fantastic. Just a few minor nitpicks, and I think we're good to go.

**Review by wren — APPROVED** [review]
🤠

**Pull request merged**